api/ServiceWorkerGlobalScope.json

```diff
@@ -0,0 +1,690 @@
+{
+  "api": {
+    "ServiceWorkerGlobalScope": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerGlobalScope",
+        "support": {
+          "webview_android": {
+            "version_added": false
```

Comment:
This should be `"40"`. 

FYI. I'm the one who originally added this information to the service worker. I was given incorrect information. Generally, the only service worker related feature that is not in Android Webview is push notifications. Please check throughout that `webview_android` has the same version number as `chrome`.